package.json

```diff
@@ -42,7 +42,7 @@
       },
       "brave-core": {
         "dir": "src/brave",
-        "branch": "master",
+        "branch": "73.0.3683.27",
```

Comment:
Is this supposed to change?

Reply:
nope but I'll remove it before I merge. I thought for CI I'd do it but apparently CI already covers this.